### PR TITLE
Add check to prevent lost settings

### PR DESCRIPTION
**Pull request by dzedro**
https://progress.opensuse.org/issues/16408

Caasp master @kravciak any objections ?

tests:
[CaaSP-plain](http://10.100.12.155/tests/5587)
[CaaSP-worker](http://10.100.12.155/tests/5588)

### CODE REVIEW

**Comment by okurz** [comment]
+1

File: tests/casp/oci_install.pm [hunk 1]
```diff
@@ -16,21 +16,56 @@ use base "y2logsstep";
...
-    send_key $cmd{install};
+
+        # Confirm installation start
+        assert_screen "startinstall";
+
+        # poo#16408 part 2
+        if ($abort) {
+            send_key 'alt-b';    # abort
+            assert_screen 'oci-overview-filled';
+            set_var 'SIMPLE_PW_CONFIRMED', 0;    # to handle_simple_pw again
```
Comment (kravciak) [inline tests/casp/oci_install.pm]: Removing set_var 'SIMPLE_PW_CONFIRMED' should work now as SR is merged in new ISO:
https://build.suse.de/request/show/128853

File: tests/casp/oci_install.pm [hunk 2]
```diff
@@ -16,21 +16,56 @@ use base "y2logsstep";
...
-    if (check_var('REGISTER', 'installation')) {
-        assert_screen 'registration-online-repos';
-        send_key "alt-y";
-    }
+        # Accept simple password
+        handle_simple_pw;
+
+        # Accept update repositories during installation
+        if (check_var('REGISTER', 'installation')) {
+            assert_screen 'registration-online-repos';
```
Comment (kravciak) [inline tests/casp/oci_install.pm]: This will fail in jobs with registration - as online repos are confirmed only once

**Comment by dzedro** [comment]
It is fixed in build 108, will update the proper behavior

File: tests/casp/oci_install.pm [hunk 3]
```diff
@@ -28,9 +46,18 @@ sub run() {
         send_key "alt-y";
```
Comment (kravciak) [inline tests/casp/oci_install.pm]: Why not use handle_simple_pw here? (you might be sending alt-y too soon)

Comment (dzedro) [inline tests/casp/oci_install.pm]: You blame you own code, which I didn't even touch ? lack of sleep again ? :P

Comment (kravciak) [inline tests/casp/oci_install.pm]: Sorry, my bad :) - LGTM, do you have PR for needles?

Comment (dzedro) [inline tests/casp/oci_install.pm]: yes

**Pull request merged**